src/pydash/strings.py

```diff
@@ -165,12 +165,86 @@
 }
 
 # Use Javascript style regex to make Lo-Dash compatibility easier.
-UPPER = "[A-Z\\xC0-\\xD6\\xD8-\\xDE]"
-LOWER = "[a-z\\xDf-\\xF6\\xF8-\\xFF]+"
-RE_WORDS = "/{upper}+(?={upper}{lower})|{upper}?{lower}|{upper}+|[0-9]+/g".format(
-    upper=UPPER, lower=LOWER
-)
+RE_WORDS = "/[^\x00-\x2f\x3a-\x40\x5b-\x60\x7b-\x7f]+/g"
```

Comment:
Wouldn't hurt to have a link back to the Lodash source where all these are pulled from (e.g. https://github.com/lodash/lodash/blob/master/.internal/unicodeWords.js and any other ones that might have been used). Will help when doing future sync-ups.

Reply:
Agree. I have added references in the commit [de757f7](https://github.com/dgilland/pydash/pull/159/commits/de757f7a0b5ec431c980df21f32edab25551a248)

Reply:
Commit de757f7 also fix the previous commit of failing checks due to the formatting.